lib/Hakyll/Web/Feed.hs

```diff
@@ -41,20 +42,20 @@ import qualified Data.Text.Encoding          as T
 
 
 --------------------------------------------------------------------------------
-rssTemplate :: String
-rssTemplate = T.unpack $
+rssTemplate :: Item String
+rssTemplate = Item "templates/rss.xml" $ T.unpack $
```

Comment:
Maybe we should call these something like `Item "_hakyll/templates/rss.xml"` so they can't conflict with actual user-provided items?

Reply:
Ah, this change is actually part of #462 on which this PR is based on :-)
I think the `Item` here is solely used for generating template error messages, it never faces the user anywhere. Or am I missing something that `Compiler` internals would do to an `Item`?